src/linter/messages.ts

```diff
@@ -250,21 +252,24 @@ export const MESSAGE_INFO = {
 		details: () => undefined,
 	},
 
-	[MESSAGE.HTML_IN_XML]: {
+	[MESSAGE.DEPRECATED_DECLARATIVE_SUPPORT]: {
 		severity: LintMessageSeverity.Error,
 		ruleId: RULES["no-deprecated-api"],
 
-		message: () => `Usage of native HTML in XML Views/Fragments is deprecated`,
-		details: () => `{@link topic:be54950cae1041f59d4aa97a6bade2d8 Using Native HTML in XML Views (deprecated)}`,
+		message: () => `DeclarativeSupport is deprecated.`,
+		details: () =>
+			`Please consider using {@link sap.ui.core.mvc.XMLView XMLViews} or` +
+			` {@link topic:e6bb33d076dc4f23be50c082c271b9f0 Typed Views} instead. For more information,` +
+			` see the documentation on {@link topic:91f27e3e6f4d1014b6dd926db0e91070 View types}.`,
 	},
 
-	[MESSAGE.LIB_INIT_API_VERSION]: {
+	[MESSAGE.DEPRECATED_LESS_SUPPORT]: {
 		severity: LintMessageSeverity.Error,
 		ruleId: RULES["no-deprecated-api"],
 
-		message: ({libInitFunction}: {libInitFunction: string}) =>
-			`Deprecated call to ${libInitFunction}(). Use parameter {apiVersion: 2} instead`,
-		details: () => `{@link sap.ui.core.Lib.init Lib.init}`,
+		message: () => `LessSupport is deprecated.`,
+		details: () =>
+			"Not required anymore! Use UI5 Tooling https://sap.github.io/ui5-tooling/stable/ to compile LESS to CSS on the fly.",
```

Comment:
I would suggest this change to align the message with the `DEPRECATED_DECLARATIVE_SUPPORT` message

```suggestion
			"Please consider using UI5 Tooling https://sap.github.io/ui5-tooling/stable/ to compile LESS to CSS on the fly.",
```